Test.C

```diff
@@ -1,53 +1,161 @@
+#include <stdio.h>
+#include <stdlib.h>
+
 /*
 
-Hello
+defining number literals
 
 */
 
-int main(){ 
-
-    //initialize game 
+#define NAME_LENGTH 20
+#define DESCRIPTION_LENGTH 200
+#define NUM_EQUIPMENT 10
+#define NUM_ENCRYPTION 10
+#define NUM_ATTACK 5
+
+int main(){
+
+    //initialize game
+
+    struct Equipment {
+        int id;
+        char name[NAME_LENGTH]; // name of the piece of Equipment
+        char description[DESCRIPTION_LENGTH]; // short description of Equipment
+        int credits; // cost of the Equipment
```

Comment:
I think `credits` here is unclear and `cost` would be more appropriate. We could also add a `sellValue` attribute, and just set it 1/2 cost, but have it there if we need more control.
```suggestion
        int cost; // cost of the Equipment
```